packages/create-schemas/src/generate.ts

```diff
@@ -20,10 +21,29 @@ export async function generate(config: ResolvedConfig): Promise<GenerationResult
         files.push(file);
     }
 
+    // ====== Load ======
+    let document: OpenAPIDocument | undefined = undefined;
+    for (const plugin of config.plugins) {
+        if (plugin.load) {
```

Comment:
A "load" hook was added to plugins because it didn't make sense for every plugin to implement their own loading behavior in their `startBuild` step.

A "load" hook is common in almost all bundlers.